x-pack/plugins/maps/server/routes.js

```diff
@@ -598,6 +600,40 @@ export async function initRoutes(
   );
 
   if (drawingFeatureEnabled) {
```

Comment:
I'd consider extracting the registration of all these routes into a separate module, similar to `initMvtRoutes`. They are all related, and start to clutter this top-level file.